nowcasting_dataset/dataset/batch.py

```diff
@@ -20,10 +21,8 @@
 
 _LOG = logging.getLogger(__name__)
 
-LOCAL_TEMP_PATH = Path("~/temp/").expanduser()
 
-
-def write_batch_locally(batch: List[Example], batch_i: int):
+def write_batch_locally(batch: List[Example], batch_i: int, path: Union[Path, Pathy]):
```

Comment:
weoth adding a default of path = Path("~/temp/").expanduser()? or is this too hiddne?

Reply:
Ah, good question.  Hmm... I think that, because `~/temp/` is set as the default in `model.py`, we perhaps don't want to set the default _twice_ (i.e. in `model.py` and here?).  But I don't have any strong feelings about this!

I have just added `path` to the docstring though (oops!)